CHANGELOG.md

```diff
@@ -35,6 +35,7 @@
   + **Property Changes**
     - AWS::AppSync::DataSource HttpConfig (__added__)
     - AWS::DAX::Cluster SSESpecification (__added__)
+    - AWS::DynamoDB::Table ServerSideEncryption (__added__)
```

Comment:
These are autogenerated, so no need to add this line here.

Reply:
I see. I will remove it in the next revision.